tests/Feature/UserTest.php

```diff
@@ -65,4 +65,30 @@ public function test_updateUser()
         $user->refresh();
         $this->assertEquals('user111', $user->name);
     }
+
+    public function test_getUserEmails_method()
+    {
+        $this->artisan('passport:install');
+        $users = User::factory()->count(3)->create();
+        $response = $this->json('GET', 'api/users')->assertStatus(200);
+        $response->assertJsonStructure([]);
```

Comment:
you didn't test emails in response. You only know that the status of the request is 200. But we don't know if the user gets emails or not.